### ISSUE TITLE
Financial disclosure tests not passing on master CI

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#1631]
Well, this is a lesson to me to never deploy straight to master, b/c you never know what's going to happen, and it's a lesson to you @flooie to make sure to add `__init__.py` files, I guess (kind of a lame lesson, but I'm not sure what else to learn from it). 

The history is that as part of doing the mypy work, it thought that `cl.disclosures` was a third party library, i.e., it thought it wasn't a part of CourtListener, and it threw an error. That was odd, but I eventually figured out that the problem was that the `cl.disclosures` lacked a `__init__.py` file, which was throwing off mypy. I added it, and fixed some other tests, but CI broke:

https://github.com/freelawproject/courtlistener/runs/2221557623?check_suite_focus=true

Looking at the failing test, I was confused, since I hadn't touched that code, and since it was reporting exactly 2× the number of results expected. Huh?:

```
======================================================================
FAIL: test_financial_disclosure_ingestion (cl.disclosures.tests.DisclosureIngestionTest)
Can we successfully ingest disclosures at a high level?
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/courtlistener/cl/disclosures/tests.py", line 50, in test_financial_disclosure_ingestion
    self.assertTrue(
AssertionError: False is not true : Should have 19 ingested investments, not 28
```

I bisected my commits to figure out which one broke it, and eventually figured out that it was 30e48bc37da1f3fca325564afdf11fff04143a14, which tweaked some hints (that wouldn't break tests, right??) and added the `__init__.py` file. 

Digging into that, I realized that the reason the test broke is because it had never run before! Previous runs in Github actions had 432 tests. After adding the init file, we have 447.

Can you please take a look and fix the test or the code or whatever?

**Comment by mlissner** [comment]
(Feel free to push to master on this one.)

**Comment by flooie** [comment]
Yep - thanks for doing the leg work on this one.

**Comment by mlissner** [comment]
Fixed via #1641
